clientlib/src/main/proto/yelp/nrtsearch/suggest.proto

```diff
@@ -147,4 +147,13 @@ message FuzzyInfixSuggester {
   int32 maxEdits = 6; //Maximum number of edits for fuzzy suggestions
   bool transpositions = 7; //Whether transpositions are allowed
   bool unicodeAware = 8; //True if all edits are measured in unicode characters, not UTF-8 bytes
+}
+
+/* The schema of suggest item to build suggest index*/
+message NrtsearchIndex {
+  int64 uniqueId = 1;
+  repeated string searchTexts = 2;
+  int64 score = 3;
+  repeated string contexts = 4;
+  bytes payload = 5;
```

Comment:
Is this the schema for a specific index or is this something generic ?

Reply:
This is generic for all suggest indices, which include category type, textual query type, and business type.